src/main/java/de/chojo/repbot/ReputationBot.java

```diff
@@ -259,6 +260,7 @@ private void initBot() {
                     }
                     log.error(LogNotify.NOTIFY_ADMIN, "Command execution of {} failed\n{}", context.command().meta().name(), context.args(), throwable);
                 })
+                .withMenuService(Consumers.empty())
```

Comment:
Why do we need an empty Consumer here?

Reply:
There was no default method without an argument for adding services. I added one and used it.